pyserini/util.py

```diff
@@ -103,3 +133,55 @@ def download_and_unpack_index(url, index_directory='indexes', force=False, verbo
     tarball.extractall(index_directory)
     tarball.close()
     os.remove(local_tarball)
+
+def download_to_cache_and_unpack_index(url, md5, force=False, verbose=True):
```

Comment:
There's a fair bit of code duplication here between this method and `download_and_unpack_index`. Wouldn't it make sense to refactor to use common code?

Isn't this method just `download_and_unpack_index` to a fixed directory (user cache) with a bit error checking? Do we need a separate code path?

Reply:
Also, can we make sure `prebuilt` is in the name to make it clear what the method does?